cleanlab/filter.py

```diff
@@ -611,26 +615,24 @@ def _keep_at_least_n_per_class(prune_count_matrix, n, *, frac_noise=1.0):
 
     Parameters
     ----------
-
     prune_count_matrix : np.array of shape (K, K), K = number of classes
         A counts of mislabeled examples in every class. For this function.
         NOTE prune_count_matrix is transposed relative to confident_joint.
 
     n : int
         Number of examples to make sure are left in each class.
 
-    frac_noise : float
-        When frac_noise = 1.0, return all estimated noise indices.
-        Value in range (0, 1] that determines the fraction of noisy example
-        indices to return based on the following formula for example class k.
-        frac_noise * number_of_mislabeled_examples_in_class_k, or
-        frac_noise * inverse_noise_rate_class_k * num_examples_s_equal_k
+    frac_noise : float, default=1.0
+      Used to only return the "top" ``frac_noise * num_label_issues``. The choice of which "top"
+      label issues to return is dependent on the `filter_by` method used. It works by reducing the
+      size of the off-diagonals of the `prune_count_matrix` of given labels and true labels
+      proportionally by `frac_noise` prior to estimating label issues with each method.
 
     Returns
     -------
-
     prune_count_matrix : np.array of shape (K, K), K = number of classes
-        Number of examples to remove from each class, for every other class."""
+        Number of examples to remove from each class, for every other class.
```

Comment:
"to remove from each class, for every other class"

- not exactly sure what this is supposed to say.